spatialos-sdk/examples/project-example/generated_code.rs

```diff
@@ -534,9 +534,9 @@ mod generated {
 
         #[derive(Debug)]
         pub struct Coordinates {
-            x: f64,
-            y: f64,
-            z: f64,
+            pub x: f64,
```

Comment:
I'll modify the code generation template to do this for everything.

Reply:
Yeah, I hand't looked into that myself since I could manually fix it for now.